test/e2e/images_test.go

```diff
@@ -435,7 +435,7 @@ RUN > file2
 		result := podmanTest.Podman([]string{"image", "prune", "--filter", "label=abc", "--force"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(result.OutputToStringArray()).To(HaveLen(1))
+		Expect(result.OutputToString()).To(BeEmpty())
```

Comment:
This is the change which is causing issues `test/e2e/images_test.go` this should be reverted to `Expect(result.OutputToStringArray()).To(HaveLen(1))`